ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -101,7 +92,12 @@ function processData(
         sorted.reverse();
     }
 
-    return sorted;
+    return {
+        complianceStandards,
+        // Slice to limit number of bars in chart.
+        // Reverse since Victory charts renders items from bottom to top
+        complianceLevelsByStandard: sorted.slice(0, 6).reverse(),
```

Comment:
oh do you know why we are limiting the number of bars in the chart? it looks like we were limiting them before but it would be great to not limit but render in a scroll area if possible

Reply:
Good question. If it becomes possible to filter out operator profiles, then limit might become a moot point.